eva-accession-remapping/src/test/java/uk/ac/ebi/eva/accession/remapping/batch/io/VariantContextWriterTest.java

```diff
@@ -74,59 +136,80 @@ public void writeRsId() throws Exception {
 
         long variantCount = forEachVcfDataLine(output, (String[] columns) -> {
             assertEquals(COLUMNS_IN_VCF_WITHOUT_SAMPLES, columns.length);
-            assertEquals("RS=rs" + rsId, columns[VCF_INFO_COLUMN]);
+            assertInfo(rsId, columns[VCF_INFO_COLUMN]);
         });
         assertEquals(1, variantCount);
     }
 
+    private void assertInfo(Long rsId, String infoColumn) {
+        assertInfo(rsId, PROJECT_ACCESSION, infoColumn);
+    }
+    private void assertInfo(Long expectedRsId, String expectedProject, String infoColumn) {
+        String[] infos = infoColumn.split(";");
+        if (expectedRsId != null) {
+            assertEquals(2, infos.length);
+            String project, rs;
+            if (infos[0].startsWith("RS=")) {
+                project = infos[1];
+                rs = infos[0];
+            } else {
+                project = infos[0];
+                rs = infos[1];
+            }
+            assertEquals("RS=rs" + expectedRsId, rs);
+            assertEquals("PROJECT=" + expectedProject, project);
```

Comment:
Should tests be added for invalid character replacement too?

Reply:
good idea